docs/release-notes.md

```diff
@@ -2,6 +2,8 @@
 
 ## Version 2.x release notes
 
+[ProxySQL 2.5.2 and ProxySQL admin tools (2023-06-12)(2.5.2.md)]
```

Comment:
[ProxySQL 2.5.2 and ProxySQL admin tools (2023-06-12)(2.5.2.md)] -> the link is broken because ] is in a wrong place

Reply:
Fixed.